swig/StringArray.hpp

```diff
@@ -138,13 +129,12 @@ class StringArray
     /**
      * Deletes the allocated strings.
      */
-    void _release_strings() noexcept
-    {
+    void _release_strings() noexcept {
         std::for_each(_array.begin(), _array.end(), [](char* c) { delete[] c; });
     }
 
     const size_t _string_size;
     std::vector<char*> _array;
 };
 
-#endif // __STRING_ARRAY_H__
+#endif  // LIGHTGBM_SWIG_STRING_ARRAY_H_
```

Comment:
```suggestion
#endif  // LIGHTGBM_SWIG_STRINGARRAY_HPP_
```

Reply:
can rename StringArray->string_array, won't change anything in MMLSpark, but that will trigger a huge git diff as we already fixed its cpp lint issues. Hence we should do this on a separate PR, +1 @StrikerRUS ?